### PR TITLE
Added main branch to yml files

### PR DESCRIPTION
**Pull request by michaelgsharp**
This adds the branch "main" to the yml files in preparation for the rename of master to main.

### CODE REVIEW

**Review by tarekgh — APPROVED** [review]

File: build/.night-build.yml [hunk 1]
```diff
@@ -8,14 +8,15 @@ pr: none
...
-# scheduled trigger, runs at UTC 8:00 every day which is midnight of GMT-8 
+# scheduled trigger, runs at UTC 8:00 every day which is midnight of GMT-8
 schedules:
 - cron: "0 8 * * *"
   displayName: Nightly Build at midnight
   branches:
     include:
     - master
-    - releases/1.0
+    - main
```
Comment (eerhardt) [inline build/.night-build.yml:18]: Can we remove `master`?

Comment (michaelgsharp) [reply build/.night-build.yml:18]: I will, but the [steps](https://github.com/dotnet/arcade/blob/main/Documentation/Projects/M2MRenaming/Master-to-Main-renaming-guide.md) explicitly say not to remove master until almost the end. Just to make sure that pipelines keep triggering correctly until the process is done. So I will create another PR to remove master when I get to that point.

**Review by eerhardt — APPROVED** [review]
LGTM

File: build/.night-build.yml (hunk 1 above)
Comment (eerhardt) [reply build/.night-build.yml:18]: Oh, I see it is still named `master` in the repo. I thought we had renamed it already.

**Pull request merged**